src/models/state.d.ts

```diff
@@ -0,0 +1,168 @@
+export interface RankDetails {
+  slug: string;
+  title: string;
+  demographic: string;
+  content_rating: string;
+  genres: number[];
+  last_chapter: number;
+  md_covers: MdCovers;
+}
+
+export interface MdCovers {
```

Comment:
The type of ` vol ` is sometimes null in the API response (check the recommendation array of ` https://api.comick.io/comic/{comic_name}/ ` ). So consider making the type string | null.